index.html

```diff
@@ -11,12 +11,26 @@
   <body>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <button onclick="showMobileMenu();" type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
       <nav id="desktop-nav">
         <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#projects">Portfolio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+      <nav id="mobile-nav" class="hide">
+        <button onclick="hideMobileMenu();" type="button" class="close">
+          <i class="fas fa-lg fa-times"></i>
+        </button>
+        <ul>
+          <li><a class="nav-link" href="#projects">Portfolio</a></li>
```

Comment:
The project requirements specify that the hamburger buttons and menu should look like this: 

![image](https://user-images.githubusercontent.com/77038610/142715957-e5ac75f1-3a3f-4a89-9473-dc55459dee93.png)

- [ ] I suggest you to align menu items on the left side and to add an underline on menu items 
I strongly suggest you to stick to [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) template and make the design of your web app meet the template ( e.g., fonts, colors, images, margins)!👍🏻